src/librustdoc/html/static/main.js

```diff
@@ -493,7 +493,7 @@ function defocusSearchBar() {
     document.addEventListener("keypress", handleShortcut);
     document.addEventListener("keydown", handleShortcut);
 
-    function resetMouseMoved(ev) {
+    function resetMouseMoved() {
         mouseMovedAfterSearch = true;
     }
 
```

Comment:
This function is trivial; does it make sense to just inline it into the `addEventListener` call?
```js
    document.addEventListener("mousemove", function() { mouseMovedAfterSearch = true; });
```

I'm not convinced that removing this parameter is a good thing - it will still be *passed* to the function, it's just ignored (which works because JS is loosely typed and allows calling any function with any number of arguments). I'd rather rename the parameter to `_ev` or something.

Reply:
Indeed!